docs/content/privacy.md

```diff
@@ -4,24 +4,24 @@ kind: operations
 weight: 50
 ---
 
-This document provides details about OPA's anonymous version reporting feature.
+This document provides details about OPA's anonymous information reporting feature.
 
 ## Overview
 
-OPA periodically reports its version to a publicly hosted, external service. The version reports only contain the OPA
-version number (e.g., v0.12.3) and a randomly generated UUID. This feature is only applicable to the `opa run` and
-`opa version` commands.
+OPA periodically reports its version, memory usage to a publicly hosted, external service. The reports contain the OPA
+version number (e.g., v0.12.3), heap usage in bytes and a randomly generated UUID. This feature is only applicable to
+the `opa run` and `opa version` commands.
 
 In case of the `opa run` command, this feature is **ON by-default** and can be easily disabled by specifying
-the `--skip-version-check` flag. When OPA is started in either the server or repl mode, OPA calls the external service
-on a best-effort basis and shares the version it's running. The time taken to execute the remote call and process the
-subsequent response from the external service does not delay OPA's start-up.
+the `--disable-telemetry` flag. When OPA is started in either the server or repl mode, OPA calls the external service
+on a best-effort basis and shares the version it's running and current memory usage. The time taken to execute the
+remote call and process the subsequent response from the external service does not delay OPA's start-up.
```

Comment:
This seems fine for now, however, in the future we could imagine reporting other anonymous usage statistics and then we'll have to go back and edit all these sentences. Perhaps we should say "OPA periodically it's version and specific anonymous runtime statistics to a publicly..." and then define what those statistics are in a bulleted list below.

Reply:
Updated the doc to include a list of reported stats.